src/components/friends/FriendListRow.tsx

```diff
@@ -64,7 +64,16 @@ export const FriendListRow = ({
         )}
       </TableTd>
       <TableTd style={{ textAlign: "right", padding: "7px 0px" }}>
-        {!isMe && (
+        {isMe ? (
```

Comment:
Maybe this condition could be removed/the buttons could be combined, and then add `style={{visibility: isMe ? "hidden" : undefined}}` and `aria-hidden={isMe ? "true" : undefined}`. That way we don't need to duplicate the variant and size and text